src/sync/sync_permission.cpp

```diff
@@ -67,6 +67,25 @@ int64_t ns_since_unix_epoch(const system_clock::time_point& point)
 
 // MARK: - Permission
 
+Permission::Permission(realm::Results& results, size_t index)
```

Comment:
To save a bit of typing, but thinking about it taking an `Object` is cleaner and makes more sense.